client/tests/e2e/specs/test_user_details_modal.ts

```diff
@@ -0,0 +1,31 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+describe('Check user details modal', () => {
+  beforeEach(() => {
+    cy.visitApp();
+    cy.login('Boby', 'P@ssw0rd.');
+    cy.get('.organization-card__manageBtn').click();
+    cy.get('.users-container').find('.user-list-item').eq(2).find('.options-button').invoke('show').click();
+    cy.get('.user-context-menu').find('.menu-list').find('ion-item').as('menuItems');
+    cy.get('@menuItems').eq(3).contains('View details').click();
+  });
+
+  afterEach(() => {
+    cy.dropTestbed();
+  });
+
+  it('Tests user details modal', () => {
+    cy.get('.ion-page').find('.ms-modal').as('modal').find('ion-header').contains('User details');
```

Comment:
Better to use `.user-details-modal` as the selector (the class you pass when you create the modal), as it will be somewhat unique, which is not guaranteed with `ms-modal`. It also means you don't need the first ion-page, just
`cy.get('.user-details-modal').as('modal')`